prac_08/KivyDemos-master/squaring.kv

```diff
@@ -0,0 +1,16 @@
+BoxLayout:
+    orientation: 'vertical'
+    TextInput:
+        id: input_number
+        text: '7'
+        font_size: 48
+        multiline: False
+    Button:
+        text: 'Square'
+        # the following line specifies the function in the app class to call when the button is pressed
+        on_press: app.handle_calculate(input_number.text)
+    Label:
+        id: output_label
+        font_size: 48
+        # change the colour to pink
+        color: (1, 0.7, 0.7, 1)
```

Comment:
This is a nice gentle shade of pink.  I went for an intense pink with (0.811, 0.184, 0.454, 1).